lib/common_components/qr_scanner.dart

```diff
@@ -80,12 +99,11 @@ class _QRScannerState extends State<QRScanner> {
     }
   }
 
-  Future<void> onScan(
-      String searchedAtsign, List<Offset> offsets, context) async {
+  Future<void> onScan(String? searchedAtsign, context) async {
     LoadingDialog().show(text: '$searchedAtsign', heading: 'Fetching');
 
     var _searchedAtsignData =
-        SearchService().getAlreadySearchedAtsignDetails(searchedAtsign);
+        SearchService().getAlreadySearchedAtsignDetails(searchedAtsign!);
```

Comment:
@Pritam-deb There's no check here, if `searchedAtsign` is null or not, I think this can break the code.